packages/DropdownMenu/src/Item.styles.js

```diff
@@ -28,7 +28,7 @@ const itemStyles = css`
 
     colorString = isDestructive ? "orangeDarken10" : "colorBlack";
     return { color: tokens.color[colorString] };
-  }}}
+  }}
```

Comment:
🤔 this got remove but not opened `{` got removed?

Reply:
Yeah there was one too many!